versions.config

```
@@ -8,7 +8,7 @@ RAW_IMAGE_VERSION="v0.2.1"
 RAW_IMAGE_CHECKSUM="0fe529a329e0fe51fa0bff0aadf63f17b30f3db6b9fa882431fe51c1ae5f9302"
```

You can try a new release 0.3.1 to see if the PARTUUID now looks better.

```suggestion
RAW_IMAGE_VERSION="v0.3.1"
RAW_IMAGE_CHECKSUM="ccff10498fb45fb76c6064988fb01b3543adfdb70ee7e5fb04b51885573684a6"
```

---

Yep, will do.  I'll test it out locally to make sure it works then update the PR.

---

Yep, the PARTUUID is now coming out properly with 0.3.1.